views/rock_picked.erb

```diff
@@ -8,5 +8,6 @@
     }
   </style>
 </head>
-<h1>You have picked Rock</h1>
-<%= "The computer has picked #{@computer.pick_weapon}" %>
+<h1><%= "You have picked #{@player_picked}" %></h1>
+<h3><%= "#{@winner}" %></h3>
```

Comment:
This line could still potentially work if you remove the # and curly parentheses as you the information is still going to be a string.